### DISCUSSION TITLE
Progetto Laboratorio Open Source Federico Castellani

### DISCUSSION THREAD

**Discussion by federico-castellani** [deneb-alpha/Lab_open_source-2023_2024#22] [Projects]
Presentazione divisa in sezioni e dimostrazione pratica del triage e triage bisect.

Sezione 1: Introduzione teorica al bisect
Sezione 2: Spiegazione di come Buovjaga esegue un triage
Sezione 3: Comandi Git per eseguire un bisect

**Comment by deneb-alpha** [comment]
Ciao @federico-castellani,
per la parte teorica, decidi pure se vuoi presentare della documentazione tua o se vuoi preparare delle slides. ti lascio libertà massima sulla modalità. 

Come dicevo anche a lezione, per l'esame preparati un bisect che hai già visto anche per evitare sorprese indesiderate.
Se possibile mi piacerebbe anche avere qualche dettaglio in più sulla tua esperienza con la community di LibreOffice. In particolar modo, sarebbe interessante capire se la documentazione esistente è chiara o se magari c'è altro da documentare (o se magari la documentazione richiede una ristrutturazione).
